Cargo.toml

```diff
@@ -816,44 +816,37 @@ enrichment-tables-benches = ["enrichment-tables-file"]
 [[bench]]
 name = "default"
 harness = false
-test = true
```

Comment:
I dropped these because `criterion` doesn't play well with `nextest` yet: https://github.com/bheisler/criterion.rs/issues/562

Reply:
I am okay with this, considering that we'll pick these up with clippy.